Assets/XML/Text/Technologies.xml

```diff
@@ -293,15 +314,19 @@
 	<TEXT>
 		<Tag>TXT_KEY_TECH_ARTISANRY</Tag>
 		<English>Artisanry</English>
-		<French>Artisanry</French>
+		<French>
+			<Text>Artisanat</Text>
+			<Gender>Male</Gender>
+			<Plural>0</Plural>
+		</French>
 		<German>Artisanry</German>
 		<Italian>Artisanry</Italian>
 		<Spanish>Artisanry</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_TECH_ARTISANRY_QUOTE</Tag>
 		<English>"The life so short, the craft so long in the learning."[PARAGRAPH:2]- Geoffrey Chaucer, Parliament of Foules</English>
-		<French>"The life so short, the craft so long in the learning."[PARAGRAPH:2]- Geoffrey Chaucer, Parliament of Foules</French>
+		<French>"Bien qu’on ait du c&#339;ur &#224; l’ouvrage, l’Art est long et le Temps est court."[PARAGRAPH:2]- Charles Baudelaire, Les Fleurs du mal</French>
```

Comment:
Same here.

Reply:
Okay, I think it makes sense here since they are basically different takes on the original Latin phrase.